### PR TITLE
docs: about page link

### PR DESCRIPTION
**Pull request by HassanBahati**
- [x] Remove the About link from the bottom of [https://nodejs.org/en/docs/guides/blocking-vs-non-blocking/ ](https://nodejs.org/en/docs/guides/blocking-vs-non-blocking/ )
- [x] Remove translations of that page [https://nodejs.org/en/docs/guides/blocking-vs-non-blocking/ ](https://nodejs.org/en/docs/guides/blocking-vs-non-blocking/ )

### CODE REVIEW

**Comment by HassanBahati** [comment]
@Trott  please review the changes in reference to [https://github.com/nodejs/nodejs.org/issues/4875#issuecomment-1279406478](https://github.com/nodejs/nodejs.org/issues/4875#issuecomment-1279406478)


Screenshot from 2022-10-15 16-50-32

**Comment by Trott** [comment]
> * [x]  Remove the About link from the bottom of [https://nodejs.org/en/docs/guides/blocking-vs-non-blocking/ ](https://nodejs.org/en/docs/guides/blocking-vs-non-blocking/)

That change looks good to me. 👍 

> * [x]  Remove translations of that page [https://nodejs.org/en/docs/guides/blocking-vs-non-blocking/ ](https://nodejs.org/en/docs/guides/blocking-vs-non-blocking/)

I'm sorry! I didn't communicate clearly about this one. I meant to just remove the same About link at the bottom of all the translations of this page, not to remove the translations entirely. Can you change this to leave the translations in place? (You can, if you wish, remove any translations that consist entirely of English and don't have any actual translated material in them, but that's optional.)

**Comment by HassanBahati** [comment]
> > * [x]   Remove the About link from the bottom of [https://nodejs.org/en/docs/guides/blocking-vs-non-blocking/ ](https://nodejs.org/en/docs/guides/blocking-vs-non-blocking/)
> 
> That change looks good to me. +1
> 
> > * [x]   Remove translations of that page [https://nodejs.org/en/docs/guides/blocking-vs-non-blocking/ ](https://nodejs.org/en/docs/guides/blocking-vs-non-blocking/)
> 
> I'm sorry! I didn't communicate clearly about this one. I meant to just remove the same About link at the bottom of all the translations of this page, not to remove the translations entirely. Can you change this to leave the translations in place? (You can, if you wish, remove any translations that consist entirely of English and don't have any actual translated material in them, but that's optional.)

Okay please, let me undo removal of entire page and only remove the about link from bottom of all translation pages

**Comment by HassanBahati** [comment]
@Trott I have made the change, please review

File: package-lock.json [hunk 1]
```diff
@@ -5,7 +5,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "nodejs.org",
```
Comment (Trott) [inline package-lock.json]: This change seems to be extraneous, I think?

**Comment by Trott** [comment]
Looks good to me other than the one line removed from `package.json` which seems unrelated to the other changes.

File: package-lock.json [hunk 2]
```diff
@@ -5,7 +5,6 @@
   "requires": true,
   "packages": {
     "": {
```
Comment (nschonni) [inline package-lock.json]:
```suggestion
    "": {
      "name": "nodejs.org",
```

**Review by nschonni — APPROVED** [review]

**Review by Trott — APPROVED** [review]

**Comment by Trott** [comment]
Thanks for the contribution! 🎉

**Comment by HassanBahati** [comment]
> Thanks for the contribution! tada

it is a pleasure...thank you @Trott for the guidance

**Pull request merged**